web/modules/weather_blocks/src/Plugin/Block/Test/EndToEnd/HourlyForecast/HourlyForecast.php.test

```diff
@@ -63,7 +63,7 @@ final class HourlyForecastStructureTest extends EndToEndBase
     }
 
     /**
-     * @group e2e2
+     * @group e2e
```

Comment:
This was just a bug.

When I'm trying to run these tests but I don't want to run the whole suite, I'll change the group name and run:
```
docker compose exec drupal phpunit --group e2e2
```

Not only is it faster, but because it's running in a single process, I get meaningful output too so I can actually understand why the tests are failing.